README.md

```diff
@@ -41,6 +41,11 @@ VVV requires recent versions of both Vagrant and VirtualBox.
 
 [Vagrant](https://www.vagrantup.com) is a "tool for building and distributing development environments". It works with [virtualization](https://en.wikipedia.org/wiki/X86_virtualization) software such as [VirtualBox](https://www.virtualbox.org/) to provide a virtual machine sandboxed from your local environment.
 
+There are also several Vagrant plugins that VVV recommends you to install:
```

Comment:
eh it should be changed there, I had to clean up a lot of these kinds of things from VVV1, I'm not surprised a handful got through